core/src/main/java/com/google/cloud/sql/ConnectorRegistry.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.sql;
+
+import com.google.cloud.sql.core.InternalConnectorRegistry;
+
+/** The public java api to control the Connector configuration. */
+public enum ConnectorRegistry {
```

Comment:
In AlloyDB the `ConnectorRegistry` implements `Closeable` and overrides:

```
@Override
  public void close() {}
```

Should we do it here so shutdown all the connectors in case the user does not do it.